lib/ui/plugins.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of dart.ui;
+
+abstract class PluginUtilities {
```

Comment:
Is it possible to write tests of these methods? If so, I think we should have some.

Reply:
Yes, it should be. I'll go ahead and write some.

Reply:
Tests added in latest commit.